demos/full/src/html.htl

```diff
@@ -25,9 +25,7 @@
 </head>
 <body>
 
-<div data-sly-list="${content.children}">
-    <div class="container-fluid">${item}</div>
-</div>
+${content.html}
```

Comment:
Note that the current state of this branch will not work: `${content.html}` is undefined. It should be changed to `${content.document.body.innerHTML}` 

(even though I'd appreciate having a shortcut like `${content.html}` if that's what most projects will need... but that's something for a separate issue I guess)